optimum/intel/neural_compressor/modeling_base.py

```diff
@@ -111,8 +123,113 @@ def _from_pretrained(
         local_files_only: bool = False,
         subfolder: str = "",
         trust_remote_code: bool = False,
+        *model_args,
         **kwargs,
     ):
+        device_map = kwargs.get("device_map", "cpu")
+        use_xpu = True if device_map == torch.device("xpu") or device_map == "xpu" else False
+
+        config = kwargs.pop("config", None)
+
+        quantization_config = kwargs.pop("quantization_config", None)
+        if not isinstance(config, PretrainedConfig):
+            config, _ = AutoConfig.from_pretrained(
+                model_id,
+                return_unused_kwargs=True,
+                **kwargs,
+            )
```

Comment:
config will be automatically loaded in https://github.com/huggingface/optimum/blob/70bd6725bceaa29316297885280a58f0587231c1/optimum/modeling_base.py#L335
```suggestion
```

Reply:
it doesn't automatically load, I debuged.

Reply:
agree, improved.